README.md

```diff
@@ -23,6 +23,15 @@ To remove the addons from your Service Catalog:
 If you update the addons and re-upload them to your web server, you need to trigger a scan by the service catalog at your XF or Kyma cluster by calling:
 `svcat sync broker helm-broker --scope cluster`
 where svcat is the [Service Catalog CLI](https://svc-cat.io/docs/cli/)
+### Development Phases
+
+#### Pushing to Branch
+If you push to a branch then the addon artifactes are uploaded to the staging location only (https://storage.googleapis.com/faros-stage-base-xf-bundles/index.yaml).
+#### Merging to master
+If you merge a branch to the master then the addon artifactes are uploaded to the staging (https://storage.googleapis.com/faros-stage-base-xf-bundles/index.yaml) and pre-production (https://storage.googleapis.com/faros-int-base-xf-bundles/index.yaml) locations.
```

Comment:
Only one of them, or?

Reply:
Just in case someone pushes to master directly?

Reply:
but that is not implmented.. when I see the Jenkinsfile there will be one upload only